server/src/main/java/org/apache/iotdb/db/qp/physical/sys/CreateMultiTimeSeriesPlan.java

```diff
@@ -164,6 +164,7 @@ public void serialize(DataOutputStream stream) throws IOException {
 
     if (alias != null) {
       stream.write(1);
+      stream.writeInt(alias.size());
```

Comment:
If we support variable length of alias array, how to know which alias match the measurement? I think we should support the same size of alias array and measurement, but the alias can be null. 

How do you think?

Reply:
You are right~I misunderstood the code here,I will fix this.
Thanks!